src/includes/getting-started-config/javascript.ember.mdx

```diff
@@ -0,0 +1,110 @@
+
+```javascript
+ENV['@sentry/ember'] = {
+  sentry: {
+    dsn: '___PUBLIC_DSN___',
+    tracesSampleRate: 1.0, // We recommend adjusting this in production
+  }
+};
+```
+
+Once this is done, all unhandled exceptions are automatically captured by Sentry.
+
+**Important:** Note your DSN. The _DSN_ (Data Source Name) tells the SDK where to send events. If you forget it, view _Settings -> Projects -> Client Keys (DSN)_ in the Sentry web UI.
+
+
+### Ember Specifics
+
+The `@sentry/ember` addon comes with some options to manage the Ember specific instrumentation, these options are set on the addon config directly. All Sentry SDK options that would be passed to `init` should be set in the `sentry` key inside your addon config instead.
+
+```javascript
+ENV['@sentry/ember'] = {
+  // Ember specific options
+  sentry: {
+    // Sentry options
+  }
+};
+```
+
+### Ember Performance
+
+The Sentry tracing integration is already setup via the Ember addon with custom Ember instrumentation for routing, components and the runloop. It sideloads `@sentry/tracing` as a chunk to instrument your application. If you would like to disable this automatic instrumentation and no longer receive the associated transactions you can set `disablePerformance` in your config.
+
+```javascript
+ENV['@sentry/ember'] = {
+  disablePerformance: true
+};
+```
+
+Performance data is transmitted using a new event type called “transactions”, which you can learn about in [Distributed Tracing](/product/performance/distributed-tracing/). To capture transactions, you must configure your SDK to set the `tracesSampleRate` configuration to a nonzero value. The default configuration will transmit 100% of captured transactions; lower this value in production to avoid consuming your quota too quickly.
+
+### Routes
+If you would like to capture timings for the `beforeModel`, `model`, `afterModel` hooks as well as `setupController` in one of your Routes, `@sentry/ember` exports a `instrumentRoutePerformance` function which can be used by replacing the default export with a wrapped Route.
+
+```javascript
+import Route from '@ember/routing/route';
+import { instrumentRoutePerformance } from '@sentry/ember';
+
+class MyRoute extends Route {
+  model() {
+    //...
+  }
+}
+
+export default instrumentRoutePerformance(MyRoute);
+```
+
+### Classic Components
+The render times of classic components are also enabled by default, with a setting to only capture render timings only above a certain duration. To change this minimum, you can modify `minimumComponentRenderDuration` in your config.
+
+```javascript
+  ENV['@sentry/ember'] = {
+    minimumComponentRenderDuration: 0, // Setting this to zero will capture all classic components.
+  };
+```
+
+To disable component instrumentation you can set `disableInstrumentComponents` in your config.
+```javascript
+ENV['@sentry/ember'] = {
+  disableInstrumentComponents: true
+};
+```
+
+### Runloop
```

Comment:
How key is this for Getting Started? Should it move to the Configuration>Basic Options page?